src/backend/columnar/sql/columnar--11.0-1--11.0-2.sql

```diff
@@ -0,0 +1,29 @@
+-- columnar--11.0-1--11.0-2.sql
+
+INSERT INTO pg_depend
+SELECT -- Define a dependency edge from "columnar table access method" ..
+       'pg_am'::regclass::oid as classid,
+       (select oid from pg_am where amname = 'columnar') as objid,
+       0 as objsubid,
+       -- ... to each object that is registered to pg_class and that lives
+       -- in "columnar" schema. That contains catalog tables, indexes
+       -- created on them and the sequences created in "columnar" schema.
+       --
+       -- Given the possibility of user might have created their own objects
+       -- in columnar schema, we explicitly specify list of objects that we
+       -- are interested in.
+       'pg_class'::regclass::oid as refclassid,
+       columnar_schema_members::regclass::oid as refobjid,
+       0 as refobjsubid,
+       'n' as deptype
+FROM unnest(ARRAY['columnar.chunk',
```

Comment:
If we create new tables in columnar schema, we will need to add them here but we can forget to add them and it would cause the same problem. Now that we know it will create similar problems, can we do something so that we don't have a missing dependency of this kind?

Reply:
I added a test to multi_extension.sql for that, so shouldn't it be fine ?

Reply:
umm, it might cover that but we will still need to add a new version of this with every additional table/sequence etc, which might be reasonable if there isn't something we can do about it